<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>general</class>
 <widget class="QWidget" name="general">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>404</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupingAndSortingGroupBox">
     <property name="title">
      <string>Grouping and sorting</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="groupingStrategyLabel">
        <property name="text">
         <string>Grouping:</string>
        </property>
        <property name="buddy">
         <cstring>groupingStrategy</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="groupingStrategy"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="sortingStrategyLabel">
        <property name="text">
         <string>Sorting:</string>
        </property>
        <property name="buddy">
         <cstring>sortingStrategy</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KComboBox" name="sortingStrategy"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filtersGroupBox">
     <property name="title">
      <string>Filters</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="showOnlyCurrentDesktop">
        <property name="text">
         <string>Only show tasks from the current desktop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showOnlyCurrentActivity">
        <property name="text">
         <string>Only show tasks from current activity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showOnlyCurrentScreen">
        <property name="text">
         <string>Only show tasks from the current screen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showOnlyMinimized">
        <property name="text">
         <string>Only show tasks that are minimized</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showOnlyTasksWithLaunchers">
        <property name="text">
         <string>Only show tasks that have launchers</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="jobsGroupBox">
     <property name="title">
      <string>Jobs</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="jobCloseModeLayout">
        <item>
         <widget class="QLabel" name="jobCloseModeLabel">
          <property name="text">
           <string>Close jobs:</string>
          </property>
          <property name="buddy">
           <cstring>jobCloseMode</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="jobCloseMode"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="connectJobsWithTasks">
        <property name="text">
         <string>Connect jobs with tasks</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="groupJobs">
        <property name="text">
         <string>Group similar jobs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
